src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -98,7 +98,6 @@ func doMain() error {
 
 	// Schedule cleanup actions
 	defer logging.Close()
-	defer pacOff()
```

Comment:
That defer only worked when the method blocked, which the new one doesn't do. I think either way is fine though.